Octokit.Tests/Models/CheckRunEventTests.cs

```diff
@@ -256,5 +256,257 @@ public void CanBeDeserialized()
+    ""archived"": false,
+    ""open_issues_count"": 3,
+    ""license"": null,
+    ""forks"": 0,
+    ""open_issues"": 3,
+    ""watchers"": 0,
+    ""default_branch"": ""main""
+  },
+  ""organization"": {
+    ""login"": ""github"",
+    ""id"": 340,
+    ""node_id"": ""MDEyOk9yZ2FuaXphdGlvbjM4MzAyODk5"",
+    ""url"": ""https://api.github.com/orgs/github"",
+    ""repos_url"": ""https://api.github.com/orgs/github/repos"",
+    ""events_url"": ""https://api.github.com/orgs/github/events"",
+    ""hooks_url"": ""https://api.github.com/orgs/github/hooks"",
+    ""issues_url"": ""https://api.github.com/orgs/github/issues"",
+    ""members_url"": ""https://api.github.com/orgs/github/members{/member}"",
+    ""public_members_url"": ""https://api.github.com/orgs/github/public_members{/member}"",
+    ""avatar_url"": ""http://alambic.github.com/avatars/u/340?"",
+    ""description"": ""How people build software.""
+  },
+  ""sender"": {
+    ""login"": ""octocat"",
+    ""id"": 5346,
+    ""node_id"": ""MDQ6VXNlcjIxMDMxMDY3"",
+    ""avatar_url"": ""http://alambic.github.com/avatars/u/5346?"",
+    ""gravatar_id"": """",
+    ""url"": ""https://api.github.com/users/octocat"",
+    ""html_url"": ""http://github.com/octocat"",
+    ""followers_url"": ""https://api.github.com/users/octocat/followers"",
+    ""following_url"": ""https://api.github.com/users/octocat/following{/other_user}"",
+    ""gists_url"": ""https://api.github.com/users/octocat/gists{/gist_id}"",
+    ""starred_url"": ""https://api.github.com/users/octocat/starred{/owner}{/repo}"",
+    ""subscriptions_url"": ""https://api.github.com/users/octocat/subscriptions"",
+    ""organizations_url"": ""https://api.github.com/users/octocat/orgs"",
+    ""repos_url"": ""https://api.github.com/users/octocat/repos"",
+    ""events_url"": ""https://api.github.com/users/octocat/events{/privacy}"",
+    ""received_events_url"": ""https://api.github.com/users/octocat/received_events"",
+    ""type"": ""User"",
+    ""site_admin"": false
+  },
+  ""installation"": {
+    ""id"": 1
+  }
+}";
+
+            var serializer = new SimpleJsonSerializer();
+
+            var payload = serializer.Deserialize<CheckRunEventPayload>(json);
+
+            Assert.Equal("rerequested", payload.Action);
+            Assert.Equal("d6fde92930d4715a2b49857d24b940956b26d2d3", payload.CheckRun.HeadSha);
+            Assert.Equal(4, payload.CheckRun.Id);
+            Assert.Equal(CheckStatus.Completed, payload.CheckRun.Status);
+            Assert.Equal(CheckConclusion.Neutral, payload.CheckRun.Conclusion);
+            Assert.Equal("dosomeaction", payload.RequestedAction.Identifier);
+            Assert.Equal(5, payload.CheckRun.CheckSuite.Id);
+            Assert.Equal(CheckStatus.Completed, payload.CheckRun.CheckSuite.Status.Value);
+            Assert.Equal(CheckConclusion.Neutral, payload.CheckRun.CheckSuite.Conclusion);
+            Assert.Equal(AccountType.Enterprise, payload.CheckRun.App.Owner.Type);
```

Comment:
This is doing the same assertions as the test above, with the addition of the `payload.CheckRun.App.Owner.Type` assertion. Open to suggestions on whether this should do just the one assertion, or if a `payload.CheckRun.App.Owner.Type == AccountType.Organization` should be added to the test above. Or perhaps adding a test at all is not worthwhile!